packages/client/src/client/App.tsx

```diff
@@ -21,8 +18,7 @@ export async function initApp() {
 
   checkTradingGatewayCompatibility()
 
-  registerFxNotifications()
-  registerCreditAcceptedNotifications()
+  registerCreditNotifications()
```

Comment:
this is where your dup notifications are coming from, btw .. this is run in all the windows